lib/project_types/script/layers/infrastructure/script_service.rb

```diff
@@ -117,7 +117,7 @@ def self.api_client(ctx, api_key, shop_domain)
               url: "https://script-service.myshopify.io/graphql",
               token: "",
               api_key: api_key,
-              shop_id: shop_domain&.to_i
```

Comment:
@jacobsteves do you recall why it was done this way in https://github.com/Shopify/shopify-app-cli/pull/584 ?

Reply:
#584 copied over the `deploy` command from script's private fork of the CLI. [Heres](https://github.com/Shopify/temp-shopify-app-cli/pull/181#issuecomment-607398150) the original PR for reference. 

TLDR, we wanted a way to deploy to our service during development without having to go through Partners, and our service only knows about `shop_id` and not `shop_domain`. The plan was to set the `shop_domain` to the shop's ID whenever we wanted to push directly to the service. This approach let us do what we want without cluttering the CLI with development convenience code

Reply:
🤔 I wonder if we can make that more explicit. We could change up the env variable to allow for the expression of the Shop ID. `export BYPASS_PARTNER_PROXY_SHOP_ID = <shop ID>`